multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakUserManager.java

```diff
@@ -82,85 +77,4 @@ protected void doRemove(String id) throws ServerException {
     eventService.publish(new BeforeUserRemovedEvent(user)).propagateException();
     userDao.remove(id);
   }
-
```

Comment:
What is the alternative to this code? what about https://openid.net/specs/openid-connect-core-1_0.html#UserInfoRequest ?

Reply:
I'll check